build/common/enums/index.ts

```diff
@@ -13,6 +13,7 @@ export function generateEnumDeclarations(
   enums: Declaration[],
   inline: boolean,
   normalize: boolean,
+  compileMembersOnly?: boolean,
```

Comment:
No need to add this as an extra paramter actually, compileMembersOnly should always be true when normalize is false, and it should always be false when normalize is true, so you can just check that parameter.

Reply:
this method is also used to generate panorama types